core/src/main/java/pl/agh/edu/model/time/Time.java

```diff
@@ -41,16 +41,13 @@ public void reset() {
 
 	public void update(float delta) {
 		if (isRunning) {
-			remaining -= delta * acceleration;
-
+			remaining -= delta * acceleration * 1000f;
```

Comment:
o ile rozumiem to jest przyspieszenie do testowania

Reply:
tak ma być nw jak według ciebie ale chociaż to piątku żeby wyogodnie było testować generowanie klientów etc